### PR TITLE
Adds morning warmup to practice power poses

### PR DESCRIPTION
**Pull request by fcaroline2020**
### Common Content?

- [ ] Block/s

### Common Theme?

- [ ] Yes

Issue number: #issue-number

### Org Content?

Module | Sprint | Page Type | Block Type

## Checklist

- [ ] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [ ] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [ ] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [ ] I have run my code to check it works
- [ ] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

### CODE REVIEW

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Oh, hrm, let me think! 🤔 

So - this is one of the most famously debunked bogus findings in the psychology replication crisis and I have strong concerns with introducing it as fact or without that context. The Ted talk is particularly problematic in this way. 

But that's ok  - it doesn't need to be true to be fun. Can you change the energiser to make it clear that there's no evidence that this is anything but a fun game? (It sounds pretty fun and energisey and I don't object to the activity itself!) 

https://www.sciencedaily.com/releases/2019/10/191001110824.htm

**Comment by fcaroline2020** [comment]
> Oh, hrm, let me think! 🤔
> 
> So - this is one of the most famously debunked bogus findings in the psychology replication crisis and I have strong concerns with introducing it as fact or without that context. The Ted talk is particularly problematic in this way.
> 
> But that's ok - it doesn't need to be true to be fun. Can you change the energiser to make it clear that there's no evidence that this is anything but a fun game? (It sounds pretty fun and energisey and I don't object to the activity itself!)
> 
> https://www.sciencedaily.com/releases/2019/10/191001110824.htm

I had to do a workshop at my previous job in 2021 that was still teaching this as a part of confident presenting! I did not realize that this was debunked 😆

**Comment by fcaroline2020** [comment]
I've added a warning but we can remove this energizer all together if we have better ones with stronger evidence

File: common-content/en/energisers/confident-presenting/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+1. Start by everyone demonstrating unconfident poses
+2. Get everyone moving with a quick stretch or power pose to break the ice and release tension. 
+ - Superman pose – feet shoulder-width apart, hands on hips, chest out.
+ - The victor - hands up in the air in a V and feet apart
+ - Any others?
+3. In a power pose of your choosing go around the room and ask everyone to say their name in the most exaggerated, enthusiastic way possible, like they’re introducing themselves to a stadium of people. 
+
+> [!WARNING]  
+> The science behind power poses is not very strong, this is only to get everyone up and moving around!
```
Comment (SallyMcGrath) [inline common-content/en/energisers/confident-presenting/index.md:25]:
```suggestion
> This is not proven science. This is just a fun game to get everyone up and moving around!
```

**Comment by SallyMcGrath** [comment]
> > Oh, hrm, let me think! 🤔
> > So - this is one of the most famously debunked bogus findings in the psychology replication crisis and I have strong concerns with introducing it as fact or without that context. The Ted talk is particularly problematic in this way.
> > But that's ok - it doesn't need to be true to be fun. Can you change the energiser to make it clear that there's no evidence that this is anything but a fun game? (It sounds pretty fun and energisey and I don't object to the activity itself!)
> > https://www.sciencedaily.com/releases/2019/10/191001110824.htm
> 
> I had to do a workshop at my previous job in 2021 that was still teaching this as a part of confident presenting! I did not realize that this was debunked 😆

Yes it keeps coming back - it popped up twice in the PD curriculum last year too. I don't mind people having fun with the power posing - it's clearly popular. But we can't pass it off as science because... not really! 😅

**Review by SallyMcGrath — APPROVED** [review]

**Pull request merged**